src/components/v2/Checkbox/index.tsx

```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import MuiCheckbox from '@mui/material/Checkbox';
+import { Icon } from 'components';
+import { useStyles } from './styles';
+
+export interface ICheckboxProps {
+  value: boolean;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const Checkbox = ({ value, onChange }: ICheckboxProps) => {
+  const styles = useStyles();
+  console.log(styles);
+  return (
+    <MuiCheckbox
+      css={styles.root}
+      checked={value}
+      onChange={onChange}
+      icon={<Icon name="checkboxBorder" />}
+      checkedIcon={<Icon name="checked" />}
+    />
```

Comment:
I don't think we should use MUI for that, because with their implementation we don't have control over the state. `checked` is only used to initialize the state of the component, after that the component handles its own state which means we'd have to react to that and effectively duplicates states. You can see that because in the story you pass `noop` to the `onChange` handler, yet clicking on the checkbox changes its state.
Given how simple the design of the component is, I'd just build it with plain HTML and CSS. It will also make it simpler to correspond to our design (no ripple effect, correct colors etc...).

Reply:
That is not correct. `noop` is passed for completeness when creating the args for the story. The prop used in the story comes from the withState decorator

Reply:
Ah right! My bad

Reply:
The hover state is blue (rgba(58, 120, 255, 1)) according to the designs which is currently set

Reply:
When the checkbox isn't hovered the border color is grey `rgb(149, 151, 161)`

Reply:
Great 👍